src/components/home/Event.jsx

```diff
@@ -1,5 +1,47 @@
-const Event = ({ start, end, location, name, descripton }) => {
-  return <div>Event</div>;
+// Color Passing In: blue -> blue, anything else -> green
+// Date Start: Month/Day/Start Time
+// Date End: End Time
+// Include AM / PM on time
+// Pass Month as ALL CAPS Three Letter Word
+
+const Event = ({ start, end, location, name, descripton, color }) => {
+  const startDate = start.split("/");
+
+  let bgcolor = "";
+  if (color === "blue") {
+    bgcolor = "#1051D3";
+  } else {
+    bgcolor = "#80C393";
+  }
+
+  return (
+    <div className=" bg-ewb-blue-100 w-4/6 inline-flex">
+      <div
+        className="text-white text-center max-w-fit w-1/4 pt-6 pb-1"
+        style={{ backgroundColor: `${bgcolor}` }}
```

Comment:
<img width="637" alt="image" src="https://github.com/acm-ucr/ewb-website/assets/73259022/4605fd26-ea3f-4ec6-8129-deb9367bea88">
 you can do something like this once you import the COLORS. Don't use style, use tailwind instead